integreat_cms/cms/templates/_form_language_tabs.html

```diff
@@ -96,6 +96,11 @@
                                         <span title="{% translate "Translation missing" %}" class="no-trans">
                                             <i icon-name="x"></i>
                                         </span>
+                                    {% elif other_status == translation_status.MACHINE_TRANSLATED %}
+                                        <span title="{% translate "Current machine translation" %}"
```

Comment:
```suggestion
                                        <span title="{% translate "Machine translated" %}"
```
I think the text should be identical to the one in the normal language tab.
If you want to change it here, please also update the text in line 18:
https://github.com/digitalfabrik/integreat-cms/blob/478afa60b99a2166e01cf814c481c6cf2d86b998/integreat_cms/cms/templates/_form_language_tabs.html#L18